src/Blue/Tools/Api/Client.php

```diff
@@ -128,17 +128,19 @@ public function get($apiPath, $queryParams = [])
      * @param $apiPath
      * @param array  $queryParams
      * @param string $data
+     * @param array $formParams
      *
      * @return ResponseInterface
      */
-    public function post($apiPath, $queryParams = [], $data = '')
+    public function post($apiPath, $queryParams = [], $data = '', $formParams = [])
```

Comment:
When would $data be used here then?

Reply:
For posting the request body.  The previous version of guzzle allowed $data to be either a string or an array of key/value pairs (for posting form fields).  In the newer version of guzzle, these parameters were separated.